samples/subsys/fs/ext2/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_LOG=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_EXT2=y
+
+CONFIG_DISK_ACCESS=y
+CONFIG_DISK_DRIVER_SDMMC=y
+
+# First block of first partition after GPT
+CONFIG_EXT2_DISK_STARTING_SECTOR=34
```

Comment:
Default configuration faults my board (bl5340_dvk_cpuapp) with a stack overflow, will find the exact Kconfig that needs adjusting and post it here, probably main stack to 4KB or so, to allow sample to run on wider range of boards